src/iosource/pcap/Source.cc

```diff
@@ -176,7 +178,32 @@ void PcapSource::OpenOffline()
 	{
 	char errbuf[PCAP_ERRBUF_SIZE];
 
-	pd = pcap_open_offline(props.path.c_str(), errbuf);
+	FILE* f = nullptr;
+	if ( props.path == "-" )
+		{
+		f = stdin;
+		}
+	else
+		{
+		if ( ! (f = fopen(props.path.c_str(), "rb")) )
```

Comment:
My version of clang-tidy complains about this construct
```
An assignment within an 'if' condition is bug-prone
```

The following form seems more readable:

```suggestion
		if ( f = fopen(props.path.c_str(), "rb"); ! f )
```

Reply:
Yeah, that looks much better, thanks!